perses/distributed/relative_setup.py

```diff
@@ -486,8 +486,8 @@ def __init__(self, topology_proposal, pos_old, new_positions, use_dispersion_cor
         self.minimize()
 
         #initialize the trajectories for the lambda 0 and 1 equilibrium simulations
-        self._lambda_zero_traj = md.Trajectory(self._lambda_zero_sampler_state.positions, self._factory.hybrid_topology)
-        self._lambda_one_traj = md.Trajectory(self._lambda_one_sampler_state.positions, self._factory.hybrid_topology)
+        self._lambda_zero_traj = md.Trajectory(np.array(self._lambda_zero_sampler_state.positions), self._factory.hybrid_topology)
```

Comment:
I'm not sure you should do this. Shouldn't `sampler_state` automatically store things in `simtk.unit.Quantity` wrapped numpy arrays? What you're doing here runs the risk of silently dropping units.

If you need to convert from Quantity-wrapped numpy arrays to nanometers for MDTraj, use an explicit unit conversion:
```python
positions / unit.nanometers
```
If sampler_state isn't storing things in Quantity-wrapped numpy arrays, let's fix that in `openmmtools.mcmc`.

Reply:
It looks like it's stored as a unit bearing list, which MDTraj apparently can't handle.

Reply:
For now, what about dealing with both contingencies (`Quantity`-wrapped `list` or `numpy.array`) with a `try...except` clause?